src/app/content/components/ContentExcerpt.spec.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import renderer from 'react-test-renderer';
+import { routes } from '..';
+import createTestStore from '../../../test/createTestStore';
+import { book } from '../../../test/mocks/archiveLoader';
+import { mockCmsBook } from '../../../test/mocks/osWebLoader';
+import MessageProvider from '../../MessageProvider';
+import { Store } from '../../types';
+import { receiveBook } from '../actions';
+import { formatBookData } from '../utils';
+import ContentExcerpt from './ContentExcerpt';
+
+describe('ContentExcerpt', () => {
+  let store: Store;
+  let consoleError: jest.SpyInstance;
+
+  beforeEach(() => {
+    store = createTestStore();
+    consoleError = jest
+      .spyOn(console, 'error')
+      .mockImplementation((msg) => msg);
+  });
+
+  afterEach(() => {
+    consoleError.mockRestore();
+  });
+
+  const render = (sourcePage: string, content: string) => renderer.create(<Provider store={store}>
+    <MessageProvider>
+      <ContentExcerpt
+        content={content}
+        sourcePageId={sourcePage}
+        className='class1'
+      />
+    </MessageProvider>
+  </Provider>);
+
+  it('fixes urls in content and matches snapshot', () => {
```

Comment:
you're testing a pretty big integration here, it would be better to mock `addTargetBlankToLinks` and `fixRelativeURLs` and assert that they are called rather than testing what they do